src/types/find-by-primary-key-options.ts

```diff
@@ -1,8 +1,7 @@
-import { TSafeOptions } from './safe-options';
-import { TSelectOption } from './select-option';
 import { TComputeOption } from './compute-option';
-import { TTransactionIsolationLevelOption } from './transaction-isolation-level-option';
 import { TParanoidOption } from './paranoid-option';
-import { TTransactionOption } from './transaction-option';
+import { TSafeOptions } from './safe-options';
+import { TSelectOption } from './select-option';
+import { TTransactionOptions } from './transaction-options';
 
-export type TFindByPrimaryKeyOptions<R, C, KR, KC> = TSafeOptions & Partial<TSelectOption<KR> & TComputeOption<KC> & TTransactionIsolationLevelOption & TParanoidOption & TTransactionOption>;
\ No newline at end of file
+export type TFindByPrimaryKeyOptions<R, C, KR, KC> = TSafeOptions & Partial<TSelectOption<KR> & TComputeOption<KC> & TTransactionOptions & TParanoidOption>;
```

Comment:
`TTransactionOption` -> `TTransactionOptions`

also ordered them the same way as other `...Options` to avoid future confusion.